src/components/FocusTrap/FocusTrapForModal/index.web.tsx

```diff
@@ -10,7 +10,7 @@ function FocusTrapForModal({children, active}: FocusTrapForModalProps) {
             focusTrapOptions={{
                 trapStack: sharedTrapStack,
                 allowOutsideClick: true,
-                fallbackFocus: document.body,
```

Comment:
According to documentation if `initialFocus` is set to `false` `fallbackFocus` option is not used:

> Setting this option to false (or a function that returns false) will prevent the fallbackFocus option from being used.